src/pages/workspace/WorkspaceProfileDescriptionPage.tsx

```diff
@@ -26,7 +26,14 @@ const parser = new ExpensiMark();
 function WorkspaceProfileDescriptionPage({policy}: Props) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
-    const [description, setDescription] = useState(() => parser.htmlToMarkdown(policy?.description ?? ''));
+    const [description, setDescription] = useState(() =>
+        parser.htmlToMarkdown(
+            policy?.description ??
+                translate('workspace.common.welcomeNote', {
+                    workspaceName: policy?.name ?? '',
+                }),
```

Comment:
I think in this case we still just want to show an empty description, not the invite note

Reply:
@narefyev91 and the description is missing from the input. 

![Screenshot 2024-03-18 at 11 13 51 at night](https://github.com/Expensify/App/assets/75031127/e30862ef-7db5-43d1-9625-e50d3c83f229)

Reply:
NVM my comment above, the doc says we'll display the default description below, so I agree that the input should be populated with the message too. 

>You have been invited to %workspaceName%! Download the Expensify mobile app at use.expensify.com/download to start tracking your expenses.